lib/demos/reference/motion_demo_shared_z_axis_transition.dart

```diff
@@ -0,0 +1,237 @@
+  const _SettingsPage();
+
+  @override
+  Widget build(BuildContext context) {
+    final localizations = GalleryLocalizations.of(context);
+
+    final settingsList = <_SettingsInfo>[
+      _SettingsInfo(
+        Icons.person,
+        localizations.demoSharedZAxisProfileSettingLabel,
+      ),
+      _SettingsInfo(
+        Icons.notifications,
+        localizations.demoSharedZAxisNotificationSettingLabel,
+      ),
+      _SettingsInfo(
+        Icons.security,
+        localizations.demoSharedZAxisPrivacySettingLabel,
+      ),
+      _SettingsInfo(
+        Icons.help,
+        localizations.demoSharedZAxisHelpSettingLabel,
+      ),
+    ];
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          localizations.demoSharedZAxisSettingsPageTitle,
+        ),
+      ),
+      body: ListView(
+        children: [
+          for (var setting in settingsList) _SettingsTile(setting),
+        ],
+      ),
+    );
+  }
+}
+
+class _SettingsTile extends StatelessWidget {
+  const _SettingsTile(this.settingData);
+  final _SettingsInfo settingData;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        ListTile(
+          leading: Icon(settingData.settingIcon),
+          title: Text(settingData.settingsLabel),
+        ),
+        const Divider(thickness: 2)
+      ],
+    );
+  }
+}
+
+class _SettingsInfo {
+  const _SettingsInfo(this.settingIcon, this.settingsLabel);
+
+  final IconData settingIcon;
+  final String settingsLabel;
+}
+
+class _RecipePage extends StatelessWidget {
+  const _RecipePage();
+
+  @override
+  Widget build(BuildContext context) {
+    final localizations = GalleryLocalizations.of(context);
+
+    final savedRecipes = <_RecipeInfo>[
+      _RecipeInfo(
+        localizations.demoSharedZAxisBurgerRecipeTitle,
+        localizations.demoSharedZAxisBurgerRecipeDescription,
+        'crane/destinations/eat_2.jpg',
+      ),
+      _RecipeInfo(
+        localizations.demoSharedZAxisSandwichRecipeTitle,
+        localizations.demoSharedZAxisSandwichRecipeDescription,
+        'crane/destinations/eat_3.jpg',
+      ),
+      _RecipeInfo(
+        localizations.demoSharedZAxisDessertRecipeTitle,
+        localizations.demoSharedZAxisDessertRecipeDescription,
+        'crane/destinations/eat_4.jpg',
+      ),
+      _RecipeInfo(
+        localizations.demoSharedZAxisShrimpPlateRecipeTitle,
+        localizations.demoSharedZAxisShrimpPlateRecipeDescription,
+        'crane/destinations/eat_6.jpg',
+      ),
+      _RecipeInfo(
+        localizations.demoSharedZAxisCrabPlateRecipeTitle,
+        localizations.demoSharedZAxisCrabPlateRecipeDescription,
+        'crane/destinations/eat_8.jpg',
+      ),
+      _RecipeInfo(
+        localizations.demoSharedZAxisBeefSandwichRecipeTitle,
+        localizations.demoSharedZAxisBeefSandwichRecipeDescription,
+        'crane/destinations/eat_10.jpg',
+      ),
+    ];
+
+    return Column(
```

Comment:
Does this need to be Column? It looks like the only non-spacing widget is the Text. It might be more simple as a Text with padding and/or centering